go/libraries/doltcore/sqle/enginetest/dolt_server_test.go

```diff
@@ -313,11 +314,21 @@ func assertResultsEqual(t *testing.T, expected []sql.Row, rows *gosql.Rows) {
 	}
 }
 
-func startServer(t *testing.T) (*sqlserver.ServerController, sqlserver.ServerConfig) {
+func startServer(t *testing.T, withPort bool, host string, unixSocketPath string) (*sqlserver.ServerController, sqlserver.ServerConfig) {
```

Comment:
godocs for this function would help, especially as we're adding many more arguments to this function.